tree-sitter-preproc/.gitignore

```diff
@@ -1,3 +1,5 @@
+Cargo.lock
```

Comment:
Why ignoring Cargo.lock here?

Reply:
Considering the hypothesis to move that grammar in an independent repository